src/Module/Admin/Logs/Settings.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Friendica\Module\Admin\Logs;
+
+use Friendica\Core\Config;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Module\BaseAdminModule;
+use Friendica\Util\Strings;
+use Psr\Log\LogLevel;
+
+class Settings extends BaseAdminModule

```

Comment:
I'm missing the file writeable check for the logfiles. Am I missing something?

Reply:
It's never been there, so yes you're missing it and no it isn't just you.